config/cable.yml

```diff
@@ -6,4 +6,5 @@ test:
 
 production:
   adapter: redis
-  url: redis://localhost:6379/1
+  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
```

Comment:
Why not `ENV.fetch("REDIS_URL", "redis://localhost:6379/1")`?

Reply:
This is config generated by Rails. So I don't know why Rails dev prefer syntax with block. May be because syntax with block is more flexible, it can accept name and do something with.

Anyway this may not be used in real life because redis on 'localhost' at production - it's possible only on VPS server. but we prefer Heroku and redis in clouds.